php-ts-mode.el

```diff
@@ -276,5 +276,9 @@ Return nil if there is no name or if NODE is not a defun node."
 
   (treesit-major-mode-setup))
 
+(when (treesit-ready-p 'php)
+  (add-to-list 'auto-mode-alist '("/\\.php_cs\\(?:\\.dist\\)?\\'" . php-ts-mode))
```

Comment:
`.php_cs` is the configuration file of the old version of PHP-CS-Fixer, so I don't think it's necessary to support it newly, but what do you think?

Please see <https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/UPGRADE-v3.md>.

Reply:
You're right. CS fixer 3 was released 2 years ago so I reckon it should be fine to remove it. I'll do it soon.